backend/entrypoint

```diff
@@ -12,7 +12,8 @@ start_backend() {
         curl -s "http://graphql:8080/v1/query" -H "Content-Type: application/json" -H "X-Hasura-Access-Key: ${HASURA_GRAPHQL_ACCESS_KEY}" --data '{"type":"replace_metadata","args":{"tables":[{"table":"view_processes","object_relationships":[],"array_relationships":[],"insert_permissions":[],"select_permissions":[{"role":"user","comment":null,"permission":{"allow_aggregations":true,"columns":["name","running","timestamp"],"filter":{}}}],"update_permissions":[],"delete_permissions":[],"event_triggers":[]},{"table":"view_bgpupdates","object_relationships":[],"array_relationships":[],"insert_permissions":[],"select_permissions":[{"role":"user","comment":null,"permission":{"allow_aggregations":true,"columns":["prefix","origin_as","peer_asn","as_path","service","type","communities","timestamp","hijack_key","handled","matched_prefix","orig_path"],"filter":{}}}],"update_permissions":[],"delete_permissions":[],"event_triggers":[]},{"table":"view_configs","object_relationships":[],"array_relationships":[],"insert_permissions":[],"select_permissions":[{"role":"user","comment":null,"permission":{"allow_aggregations":true,"columns":["raw_config","comment","time_modified"],"filter":{}}}],"update_permissions":[],"delete_permissions":[],"event_triggers":[]},{"table":"view_hijacks","object_relationships":[],"array_relationships":[],"insert_permissions":[],"select_permissions":[{"role":"user","comment":null,"permission":{"allow_aggregations":true,"columns":["active","comment","configured_prefix","hijack_as","ignored","key","mitigation_started","num_asns_inf","num_peers_seen","peers_seen","peers_withdrawn","prefix","resolved","seen","time_detected","time_ended","time_last","time_started","timestamp_of_config","type","under_mitigation","withdrawn","outdated"],"filter":{}}}],"update_permissions":[],"delete_permissions":[],"event_triggers":[]}],"query_templates":[],"remote_schemas":[]}}' > /dev/null
     fi
     service cron start
-    redis-server /etc/redis/redis.conf --daemonize yes
+    sed -i 's/bind 127.0.0.1/# bind 127.0.0.1/g' /etc/redis/redis.conf
+    redis-server /etc/redis/redis.conf --daemonize yes --protected-mode no
```

Comment:
protected-mode is off but this is fine because we are in a closed environment and no ports are exposed in the production